.gitignore

```diff
@@ -50,3 +50,6 @@ _sandbox
 env
 venv
 .python-version
+
+# IDEs
+/.idea/
```

Comment:
This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). Can you revert this please?

Reply:
Sure, no problem